src/Math-Complex/PMComplex.extension.st

```diff
@@ -17,20 +17,20 @@ PMComplex >> productWithVector: aVector [
 	^ aVector collect: [ :each | each * self ]
 ]
 
-{ #category : #'*Math-Complex' }
-PMComplex class >> random [
-		"Answers a random number with abs between 0 and 1." 
-
-	^ self abs: 1.0 random arg: 2 * Float pi random
-]
-
 { #category : #'*Math-Complex' }
 PMComplex >> random [
 	"analog to Number>>random. However, the only bound is that the abs of the produced complex is less than the length of the receive. The receiver effectively defines a disc within which the random element can be produced."
 	^ self class random * self
 	
 ]
 
+{ #category : #'*Math-Complex' }
+PMComplex class >> random [
```

Comment:
Is this used anywhere or has it been added to stay polymorphic to `Number`?

Reply:
Ah, this one is strange
I didn't add it
And if you look at the diff, the same method is marked as added and removed
Nothing was changed

So I think it's a bug in how Github calculates the diff